Formula/rpm.rb

```diff
@@ -27,51 +13,40 @@ class Rpm < Formula
     sha256 "3a276b6a3f7273f8f88effbb057e8d65fc53fa9e812054951bfdd5ce618f54ac" => :yosemite
   end
 
-  depends_on "rpm2cpio" => :build
+  depends_on "beecrypt"
   depends_on "berkeley-db"
   depends_on "libmagic"
+  depends_on "libarchive"
   depends_on "popt"
-  depends_on "libtasn1"
+  depends_on "gnupg"
   depends_on "gettext"
+  depends_on "lua"
   depends_on "xz"
-  depends_on "ossp-uuid"
+  depends_on "zstd"
+  depends_on "pkg-config" => :run
 
   def install
     # only rpm should go into HOMEBREW_CELLAR, not rpms built
-    inreplace "macros/macros.in", "@prefix@", HOMEBREW_PREFIX
+    inreplace "macros.in", "@prefix@", HOMEBREW_PREFIX
```

Comment:
Still no option to set these with configure? :(

Reply:
Not yet, but I'm looking into fixing this for a future release.